pdk/1.2/modules/ROOT/pages/policies-pdk-configure-features-grpc-request.adoc

```diff
@@ -0,0 +1,221 @@
+= Performing a Unary gRPC Call
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: To view an example policy project that makes gRPC calls, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/main/simple-oauth-2-grpc-validation/README.md[Simple OAuth 2.0 gRPC Validation Policy Example^].
+
+To interact with external services, a policy can use gRPC calls. You can only make gRPC calls from the request filter or the response filter.
+
+Custom policies only support unary gRPC calls. Policies do not support gRPC streaming.
+
+To perform a gRPC call, either inject the `GrpcClient` into the `#[entrypoint]` function or the wrapper functions:
```

Comment:
```suggestion
Depending on where you inject the `GrpcClient`, you can make calls from the request, response, or both filters.
```
I'm suggesting this to clarify a bit the different options here. If you agree with this edit, apply also the next two comments that introduce the example code.